packages/cms/src/components/cards/GeneratorCard.jsx

```diff
@@ -202,17 +208,20 @@ class GeneratorCard extends Component {
                 {localeSecondary &&
                   <LocaleName>{localeDefault}</LocaleName>
                 }
-                <VarTable dataset={displayData} />
+                <VarTable dataset={displayData} dupes={dupes}/>
               </div>
 
               {localeSecondary &&
                 <div className="cms-card-locale-container">
                   <LocaleName>{localeSecondary}</LocaleName>
-                  <VarTable dataset={secondaryDisplayData} />
+                  <VarTable dataset={secondaryDisplayData} dupes={dupes} />
                 </div>
               }
             </div>
           }
+          {dupes.length > 0 && 
+            <div style={{color: "red"}}>Warning: Variables reused!</div>
+          }
```

Comment:
@perpetualgrimace I would love to get your design review of this PR, maybe with Jimmy in person? 

Also, what do you think the text should say? I think we need to be more explicit here, something like "Variables exported by this generator conflict with another generator". Maybe it's just a little warning icon that you can see this message on hover?

Reply:
@davelandry Jimmy Jam was attempting to add a tooltip but there was some funkiness with it repeatedly opening/closing itself. I think this will need some design time to come up with a non-clunky solution.

In the meantime, I agree that the message is a little vague, and an even better one might be something like `Warning: Variables ${ dupes.split() } conflict with another generator`

Reply:
I fixed the mixin below, and changed the wording. It now looks like this:

![image](https://user-images.githubusercontent.com/1714292/70185136-3c9d7600-16b7-11ea-9b0d-072ca47f9751.png)

I don't think there's need for `dupes.join()` because they are already highlighted in the VarTable.

@davelandry do we want to push this through as-is, or should we have @perpetualgrimace take the required design time for a better solution

Reply:
@jhmullen I'm okay with no tooltips and not using `dupes.join()`, but that highlighting is waaaaay too subtle. @perpetualgrimace what do you think?

Reply:
@jhmullen 

`<p className="cms-card-error u-font-xxs">{warningText}</p>`

`.cms-card-error { @mixin error-text; }`